src/Roots.jl

```diff
@@ -13,30 +16,312 @@ import ClimaLSM:
     auxiliary_vars,
     initialize,
     initialize_auxiliary
-export RootsModel
+export RootsModel,
+    compute_flow,
+    theta_to_p,
+    p_to_theta,
+    RootsPrescribedExchange,
+    RootsParameters
+
+"""
+    RootsParameters{ft <: AbstractFloat}
+
+
+A struct for holding parameters of the Root Model. Eventually to be used with ClimaParameters.
+"""
+struct RootsParameters{ft <: AbstractFloat}
+    a_root::ft
```

Comment:
It's standard in CliMA and, from what I've seen, a general Julia convention, to capitalize the parametric type identifiers e.g. `FT` rather than `ft`. We do this elsewhere in ClimaLSM, too

Reply:
Same comment for `RootsModel` struct definition.

Reply:
got it! will fix the other instances too.